projects/view/client/rtProject/rtProjectInfoSection.html

```diff
@@ -0,0 +1,45 @@
+<template name="rtProjectInfoSection">
```

Comment:
This is about consistency. The project has started with two spaces per level of indentation. It is important that code uses consistent style, as if one person wrote the code.